minari/dataset/minari_storage.py

```diff
@@ -403,3 +404,25 @@ def _add_episode_to_group(episode_buffer: Dict, episode_group: h5py.Group):
             episode_group.create_dataset(
                 key, data=data, dtype=dtype, chunks=True, maxshape=(None, *dshape)
             )
+
+
+def get_dataset_size(dataset_id: str):
```

Comment:
Okay, last thing before merging I think
Can you make this a class function of MInariStorage, rename it to `get_size` and remove the dataset_id arg?
So you can get the size using `storage.get_size()`

You can get the data_path from the MinariStorage attribute.